h/nipsa/__init__.py

```diff
@@ -11,7 +10,7 @@ def includeme(config):
                           'memex.events.AnnotationTransformEvent')
 
     # Register the NIPSA service
-    config.register_service_factory(services.nipsa_factory, name='nipsa')
+    config.register_service_factory('.services.nipsa_factory', name='nipsa')
```

Comment:
This is necessary to prevent an import cycle, because:

- `h.tasks.nipsa` imports `h.nipsa.search`
- `h.nipsa` imports `h.nipsa.services`
- `h.nipsa.services` imports `h.tasks.nipsa`

Switching to a dotted-string import here fixes the issue, but it's worth noting that this would also be solved by moving all services into a `services` package. I expect we'll do that soon, too...